tools/readme.md

```diff
@@ -8,25 +8,28 @@ This tool will re-deploy your resources using the assets generated by Azure Dev
 3. If required, deploys an ingress controller to provide a public endpoint to access the service over the internet.
 
 ## Prerequisites
-Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash [cloudshell](https://shell.azure.com/ ) for a seamless experience.
+Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash [cloudshell](https://shell.azure.com/) for a seamless experience.
 1. [azure cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
 2. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 3. [docker](https://docs.docker.com/engine/install/)
 4. curl
 5. gunzip
 6. tar
 
+**Note:** When using Azure Cloud Shell, only Azure Container Registry is available for hosting docker images.
```

Comment:
Let's add this to the documentation also

Reply:
Yes, once I check it in, I will coordinate with Nick and Gordon to get it updated!